source/index.html.md

```diff
@@ -7,7 +7,7 @@ weight: 1
 
 This catalogue is for APIs published by public sector organisations in the UK.
 
-If you're working on an API that publishes open data or connects government with users from other organisations or wider industry, we’d like to invite you to publish details of it in this catalogue. You can publish details of your API by [creating an issue in the API Catalogue GitHub repo](https://github.com/alphagov/api-catalogue/issues).
+If you're working on an API that publishes open data or connects government with users from other organisations or wider industry, we’d like to invite you to publish details of it in this catalogue. You can publish details of your API by [creating an issue in the API Catalogue GitHub repo](https://github.com/co-cddo/api-catalogue/issues/new?assignees=&labels=enhancement&template=new-catalogue-entry-request.md&title=%5BName+of+your+department%2Forganisation%3Aname+of+your+API%5D).
```

Comment:
This does have the risk of breaking over time i.e. if we change the filename `new-catalogue-entry-request.md`.

Did we alternatively just want to go to https://github.com/co-cddo/api-catalogue/issues/new/choose ?